Description: Add shebang to Python MapScript example scripts.
Author: Bas Couwenberg <sebastic@debian.org>
Bug: https://github.com/mapserver/mapserver/issues/5773
Forwarded: https://github.com/mapserver/mapserver/pull/5774
Applied-Upstream: https://github.com/mapserver/mapserver/commit/a7f6587d8e16c090e07365cfc14a6867813a3b99

--- a/mapscript/python/examples/project_csv.py
+++ b/mapscript/python/examples/project_csv.py
@@ -1,3 +1,5 @@
+#!/usr/bin/env python
+
 """
 Simple example to read a csv file and reproject point x/y data
 
--- a/mapscript/python/examples/shpdump.py
+++ b/mapscript/python/examples/shpdump.py
@@ -1,3 +1,5 @@
+#!/usr/bin/env python
+
 """
 Dump the contents of the passed in Shapefile
 
--- a/mapscript/python/examples/shpinfo.py
+++ b/mapscript/python/examples/shpinfo.py
@@ -1,3 +1,5 @@
+#!/usr/bin/env python
+
 """
 Extracts basic descriptive information from a Shapefile
 
--- a/mapscript/python/examples/wxs.py
+++ b/mapscript/python/examples/wxs.py
@@ -1,3 +1,5 @@
+#!/usr/bin/env python
+
 """
 Output WMS GetCapabilities response for a Mapfile
 
